rclcpp/include/rclcpp/executor.hpp

```diff
@@ -195,6 +199,24 @@ class Executor
   }
 
   static void
+  execute_intra_process_subscription(
+    rclcpp::subscription::SubscriptionBase::SharedPtr & subscription)
```

Comment:
But it's ok though, and the other functions do it this way. I'd prefer to have a different pull request fix all the functions if this is really better.